spring/src/main/java/io/cloudevents/spring/http/CloudEventHttpUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudevents.spring.http;
+
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import io.cloudevents.CloudEventContext;
+import io.cloudevents.core.CloudEventUtils;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.cloudevents.core.message.MessageReader;
+import io.cloudevents.http.HttpMessageFactory;
+import io.cloudevents.http.impl.HttpMessageWriter;
+
+import org.springframework.http.HttpHeaders;
+
+/**
+ * Miscellaneous utility methods to assist with Cloud Events in the context of Spring Web
+ * frameworks. Primarily intended for the internal use within Spring-based frameworks or
+ * integrations.
+ *
+ * @author Dave Syer
+ * @since 2.0
+ */
+public class CloudEventHttpUtils {
```

Comment:
This class is usually called `*MessageFactory` in other modules (it's also final and with empty private constructor). Can we rename it to `CloudEventHttpMessageFactory`?

Reply:
I guess. But it isn't just a message reader/writer factory.

Reply:
I put in the private constructor.